database/schema.sql

```diff
@@ -0,0 +1,41 @@
+CREATE TABLE item (
+  id INT GENERATED BY DEFAULT AS IDENTITY PRIMARY KEY,
+  publish_date DATE NOT NULL,
+  archived BOOLEAN NOT NULL    
+);
+
+CREATE TABLE IF NOT EXISTS author (
+  id INT GENERATED BY DEFAULT AS IDENTITY PRIMARY KEY,
+  first_name text NOT NULL,
+  last_name text NOT NULL
+);
+
+CREATE TABLE IF NOT EXISTS game (
+  id INT GENERATED BY DEFAULT AS IDENTITY PRIMARY KEY,
+  multiplayer text NOT NULL,
+  last_played_at date NOT NULL,
+  item_id integer REFERENCES item(id) ON DELETE CASCADE
+);
+
+
+CREATE TABLE book (
+  id INT GENERATED BY DEFAULT AS IDENTITY PRIMARY KEY ,
+  publisher VARCHAR(255) NOT NULL,
+  cover_state VARCHAR(25) NOT NULL  
+);
+
+CREATE TABLE label (
+  id INT GENERATED BY DEFAULT AS IDENTITY PRIMARY KEY,
+  title VARCHAR(255) NOT NULL,
+  color VARCHAR(100) NOT NULL
+);
+
+ALTER TABLE book ADD CONSTRAINT fk_item FOREIGN KEY (id) REFERENCES item(id) ON DELETE CASCADE;
+
+ALTER TABLE label ADD CONSTRAINT fk_item FOREIGN KEY (id) REFERENCES book(id) ON DELETE CASCADE;
+
+ALTER TABLE item ADD CONSTRAINT fk_item FOREIGN KEY (id) REFERENCES book(id) ON DELETE CASCADE;
+
+ALTER TABLE item ADD CONSTRAINT fk_item FOREIGN KEY (id) REFERENCES game(id) ON DELETE CASCADE;
+
+ALTER TABLE item ADD CONSTRAINT fk_item FOREIGN KEY (id) REFERENCES author(id) ON DELETE CASCADE;
```

Comment:
![image](https://user-images.githubusercontent.com/30554633/222430636-3b003086-bc87-4ddf-9f19-aa4db486e707.png)

I get the above errors when I run the codes
I think you can use ALTER TABLE book ADD FOREIGN KEY (id) REFERENCES item(id) ON DELETE CASCADE for the first query and then replicate it for the others

Reply:
![book](https://user-images.githubusercontent.com/108219288/222435181-bd60a64f-2620-49bb-b03c-98b4fd545f82.png)
![item](https://user-images.githubusercontent.com/108219288/222435186-d5d15448-c2e1-46fb-a85d-ae2cd49697c0.png)
![label](https://user-images.githubusercontent.com/108219288/222435191-a757f167-9096-411b-be3f-5d5f0c822681.png)


The table is generated fine. Please join a call if needed